components/alert/alert.css

```diff
@@ -0,0 +1,65 @@
+span {
+	font-weight: 900;
+}
+
+.alert {
+	border-radius: 0.5rem;
+	padding: 1.1rem;
+	margin: 1rem 0;
+}
+
+.alert-success {
+	background-color: #84f1c9;
+	color: #1b7052;
+}
+
+.alert-info {
+	background-color: #eaeaec;
+	color: #696b79;
+}
+
+.alert-error {
+	color: #dc3545;
+	background-color: #fcc5c5;
+}
+
+.alert-warning {
+	color: #ff9204;
+	background-color: #ffe7c5;
+}
```

Comment:
I would suggest using the `root` element to create CSS variables for colours like this 
```
:root {
       --success-background-color: #84f1c9;
       --success-color: #1b7052;
}
.alert-success {
	background-color: var(--success-background-color);
	color: var(--success-color);
}
```
You can follow the same pattern for the rest of the alert components